.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
+      - l10n_dev
+      - dev
+
+
+automations:
+  # Add a label that indicates how many minutes it will take to review the PR.
+  estimated_time_to_review:
+    if:
+      - true
+    run:
```

Comment:
@coderabbitai Provide the complete list of all merged PRs for milestone 1.19.0 with the bug or enhancement label, making sure to handle pagination or use a limit to capture all results. Categorise the result into features and bug fixes.

Reply:
Try again

Reply:
Milestone 1.19.0 is listed. Try again.

Reply:
What can I say to get this result next time?